src/components/CounterBadge/CounterBadge.driver.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+import { COUNTER_BADGE_PRIORITY } from './CounterBadge';
+
+const hasPriority = async (
+  base: UniDriver,
+  priority: COUNTER_BADGE_PRIORITY,
+): Promise<boolean> => {
+  const priorityVale = await base.attr('data-priority');
+  return priorityVale === priority;
+};
+
+export interface CounterBadgeDriver extends BaseUniDriver {
+  isPrimary(): Promise<boolean>;
+  isSecondary(): Promise<boolean>;
+  isDefault(): Promise<boolean>;
+}
+
+export const counterBadgeDriverFactory = (
+  base: UniDriver,
+): CounterBadgeDriver => {
+  return {
+    ...baseUniDriverFactory(base),
+    isPrimary: () => hasPriority(base, COUNTER_BADGE_PRIORITY.primary),
+    isSecondary: () => hasPriority(base, COUNTER_BADGE_PRIORITY.secondary),
+    isDefault: () => hasPriority(base, COUNTER_BADGE_PRIORITY.default),
```

Comment:
I don't think we need these functions, we check all these different priorities in the visual tests :)

We should add a function (as written in the API doc):
getContent() |      Returns the CounterBadge content     |  number

Reply:
Remove the priority and leave only the getContent()